### ISSUE TITLE
int() argument must be a string, a bytes-like object or a number, not 'dict'

### ISSUE THREAD

**Issue by pmantaj** [googleapis/python-monitoring#105]
#### Environment details

  - OS type and version: `macOS Big Sur`
  - Python version: `Python 3.6.10`
  - pip version: `21.0.1`
  - `google-cloud-monitoring` version: `2.2.0`

#### Steps to reproduce

  1. Just use the following import `from google.cloud import monitoring_v3` in any file.

#### Code example

```python
from google.cloud import monitoring_v3
```

#### Stack trace
```
  File "venv/lib/python3.6/site-packages/google/cloud/monitoring_v3/__init__.py", line 18, in <module>
    from .services.alert_policy_service import AlertPolicyServiceClient
  File "venv/lib/python3.6/site-packages/google/cloud/monitoring_v3/services/alert_policy_service/__init__.py", line 18, in <module>
    from .client import AlertPolicyServiceClient
  File "venv/lib/python3.6/site-packages/google/cloud/monitoring_v3/services/alert_policy_service/client.py", line 35, in <module>
    from google.cloud.monitoring_v3.services.alert_policy_service import pagers
  File "venv/lib/python3.6/site-packages/google/cloud/monitoring_v3/services/alert_policy_service/pagers.py", line 29, in <module>
    from google.cloud.monitoring_v3.types import alert
  File "venv/lib/python3.6/site-packages/google/cloud/monitoring_v3/types/__init__.py", line 18, in <module>
    from .alert import AlertPolicy
  File "venv/lib/python3.6/site-packages/google/cloud/monitoring_v3/types/alert.py", line 21, in <module>
    from google.cloud.monitoring_v3.types import common
  File "venv/lib/python3.6/site-packages/google/cloud/monitoring_v3/types/common.py", line 51, in <module>
    class ServiceTier(proto.Enum):
  File "venv/lib/python3.6/site-packages/proto/enums.py", line 72, in __new__
    cls = super().__new__(mcls, name, bases, attrs)
  File ".pyenv/versions/3.6.10/lib/python3.6/enum.py", line 201, in __new__
    enum_member = __new__(enum_class, *args)
TypeError: int() argument must be a string, a bytes-like object or a number, not 'dict'
```

The problem is in the file: `google/cloud/monitoring_v3/types/common.py` and class `ServiceTier(proto.Enum)`. When I remove the property `_pb_options = {"deprecated": True}` everything works correctly.

Thanks!

**Comment by Atixx** [comment]
Had this same issue when upgrading to 2.2.0 from 2.0.1

**Comment by busunkim96** [comment]
Ah apologies, I missed updating the minimum required version of `proto-plus`. If you update `proto-plus` to the latest release (anything >=1.15.0 will work) the error should disappear. 

The good news is that we're starting to explicitly test the lower bound (see #102) so this should not happen in the future.

**Comment by busunkim96** [comment]
Closed by #102

**Comment by busunkim96** [comment]
Fix released in 2.2.1
